addons/web/static/src/js/core/utils.js

```diff
@@ -413,6 +604,19 @@ var utils = {
             }
         }
     },
+    /**
+     * Replace diacritics character with ASCII character
+     *
+     * @param {string} str diacritics string
+     * @param {boolean} casesensetive
+     * @returns {string} ASCII string
+     */
+    unaccent: function (str, casesensetive=false) {
```

Comment:
@mgh-odoo @switch87 Default function parameter is an ES6 feature, we can't use it (because of ie 11)

Reply:
@aab-odoo 
for the quick merge, I've created PR based on your suggestion. could you please review and merge it?
PR: https://github.com/odoo/odoo/pull/30751

@switch87 @msh-odoo 
Thanks!

Reply:
:+1:

Reply:
Thanks @aab-odoo for drawing attention